src/selectors/vaccine.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2021
+ */
+
+export const selectScannedSensorAddresses = ({ state }) => {
+  const { scannedSensorAddresses } = state;
+  return scannedSensorAddresses;
+};
```

Comment:
I used this while testing, as I was using the conditional for `isScanning`. The `state` property won't exist on the top level state now that it has been added as `vaccine` :) 

```suggestion
export const selectScannedSensors = ({ vaccine }) => {
  const { scannedSensorAddresses = [] } = vaccine || {};
  return scannedSensorAddresses;
};

export const selectIsScanning = ({ vaccine }) => {
  const { isScanning = false } = vaccine || {};
  return isScanning;
};
```

Reply:
Ahh oops sorry about that. Thanks for the tweak 😅